File: sentimental-cash/cash.py
import cs50


def main():
    while True:
        n = cs50.get_float("change: ")
        if n >= 0:
            break
    cash(n)


def cash(n):
    change = round(n * 100)
    coins = 0
    while (change > 0):
        if change >= 25:
            change -= 25
            coins += 1
        elif change >= 10:
            change -= 10
            coins += 1
        elif change >= 5:
            change -= 5
            coins += 1
        elif change >= 1:
            change -= 1
            coins += 1
    print(coins)


main()

File: cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int change;
    int coins = 0;
    do
    {
        change = get_int("Change owed: ");
    }
    while (change < 0);
    while (change > 0)
    {
        if (change - 25 >= 0)
        {
            change -= 25;
            coins++;
        }
        else if (change - 10 >= 0)
        {
            change -= 10;
            coins++;
        }
        else if (change - 5 >= 0)
        {
            change -= 5;
            coins++;
        }
        else if (change - 1 >= 0)
        {
            change -= 1;
            coins++;
        }
    }
    printf("%d\n", coins);
}
